src/components/MermaidDash.js

```diff
@@ -392,6 +394,30 @@ class MermaidDash extends Component {
     return new Set(result).size;
   }
 
+  calDistance(item1, item2) {
+    return turfDistance(point(item1), point(item2), { units: 'meters' });
+  }
+
+  getUniqueSiteCount(array) {
+    let duplicateCount = 0;
+    const totalSitesCount = array.length;
+    const coordinatesArr = array.map(({ geometry: { coordinates } }) => {
+      return coordinates;
+    });
+
+    for (let i = 0; i < totalSitesCount; i++) {
+      if (i !== totalSitesCount - 1) {
+        for (let y = i + 1; y < totalSitesCount; y++) {
+          if (this.calDistance(coordinatesArr[i], coordinatesArr[y]) < 150) {
```

Comment:
I suggest adding `// Assumes coordinates that are within 150m of each other are duplicate` for clarity just before this line